helm/prometheus-meta-operator/templates/prometheus-rules/network.all.rules.yml

```diff
@@ -75,3 +75,37 @@ spec:
         team: rocket
         {{- end }}
         topic: network
+    - alert: HighNumberOfAllocatedSockets
+      annotations:
+        description: '{{`Over 90 percent of sockets are allocated on {{ or $labels.pod_name $labels.instance }}.`}}'
```

Comment:
Can you explain the pod name OR instance logic? I'm guessing the pod name would be the node exporter pod, but wouldn't this metric always be at the node level?

Reply:
Fixed.
Thanks for pointing this out.

Reply:
Fixed.
Thanks for pointing this out.